documentation/src/test/java/guides/TestSubscribersTest.java

```diff
@@ -51,4 +51,37 @@ void failing() {
                 .assertFailedWith(IOException.class, "Boom");
         // </failing>
     }
+
+    @Test
+    void uniFunction() {
+        // <uni-function>
```

Comment:
This is a marker for embedding in the website documentation.

Reply:
Yep, from you comment, this should not end up there. I'll remove it

Reply:
@jponge So what do you recommend to test partially?

I'm using only `uni` at the moment, but could add to multi, if this one goes through